src/merc.h

```diff
@@ -85,7 +86,6 @@ typedef struct shop_data SHOP_DATA;
 typedef struct time_info_data TIME_INFO_DATA;
 typedef struct weather_data WEATHER_DATA;
 /* Rohan's finger caching struct */
```

Comment:
--comment

Reply:
I felt bad taking them out .. but yeah. The time has come :)

Reply:
oh, you meant for finger_caching...yeah done!